### PR TITLE
Grails4 upgrade

### PR DESCRIPTION
**Pull request by Rita-C**

### CODE REVIEW

**Review by nickdos — APPROVED** [review]

File: grails-app/services/au/org/ala/specieslist/HelperService.groovy [hunk 1]
```diff
@@ -126,33 +135,41 @@ class HelperService {
     }
 
     def updateDataResourceForList(drId, map) {
-        if(grailsApplication.config.collectory.enableSync?.toString()?.toBoolean()){
+        if (grailsApplication.config.collectory.enableSync?.toString()?.toBoolean()){
```
Comment (nickdos) [inline grails-app/services/au/org/ala/specieslist/HelperService.groovy:138]: may as well use `grailsApplication.config.getProperty('collectory.enableSync', Boolean, false)` here

File: grails-app/services/au/org/ala/specieslist/NameExplorerService.groovy [hunk 2]
```diff
@@ -0,0 +1,109 @@
+package au.org.ala.specieslist
```
Comment (nickdos) [inline grails-app/services/au/org/ala/specieslist/NameExplorerService.groovy:1]: Missing Copyright statement

**Review by nickdos — APPROVED** [review]

File: grails-app/services/au/org/ala/specieslist/NameExplorerService.groovy (hunk 2 above)
Comment (Rita-C) [reply grails-app/services/au/org/ala/specieslist/NameExplorerService.groovy:15]: updated

File: grails-app/services/au/org/ala/specieslist/HelperService.groovy (hunk 1 above)
Comment (Rita-C) [reply grails-app/services/au/org/ala/specieslist/HelperService.groovy]: Good idea, updated

**Pull request merged**